### DISCUSSION TITLE
Toward v5

### DISCUSSION THREAD

**Discussion by EdamAme-x** [honojs/hono#3000] [General]
It may be too early to tell, but do you have any ideas?
And this is the memorable 3000th.

**Comment by yusukebe** [comment]
Hi @EdamAme-x 

It's not a bad idea to discuss version 5. But, I've never created a "road map" intentionally. For this, I have some reasons. One is that it may add pressure on that matter to me. I have to think about them, though, of course, I know we can't implement all of the ideas. Sometimes, the "thinking" will tire me.

What I'm trying to say is that it's too early, and I'd like you to wait a bit.

**Reply by NicoPlyley** [reply]
I agree with the roadmap. An internal one may be great, but for example, there have been projects I wanted to use, once I saw their roadmap I was waiting for that feature. It's been the "top focus" for two years now and still nothing. It makes me not want to use that product as it seems untouched, even though the development is still active. 

For large projects people get excited about I think it's great, smaller ones where it all relies on a few developers I can see it being a burden.
